common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -454,7 +457,8 @@ public ICacheRecord save(@NonNull final AccountRecord accountRecord,
     @Override
     public ICacheRecord load(@NonNull final String clientId,
                              @Nullable final String target,
-                             @NonNull final AccountRecord account) {
+                             @NonNull final AccountRecord account,
+                             @NonNull final AbstractAuthenticationScheme authScheme) { // TODO implement!
```

Comment:
do we still need this todo?

Reply:
Nope this is done, will remove

Reply:
Resolved in 89e92905f760044f89e4214b3ddf00814b8e2380